sympy/polys/densebasic.py

```diff
@@ -182,6 +182,36 @@ def dmp_degree(f, u):
     else:
         return len(f) - 1
 
+def dmp_power(f, u):
+    """
+    Return the powers of variable in ``f`` in ``x_0`` in ``K[X]``.
+
+    Note that the power of 0 is zero (the SymPy object S.Zero).
+
+    Examples
+    ========
+
+    >>> from sympy.polys.domains import ZZ
+    >>> from sympy.polys.densebasic import dmp_power
+
+    >>> dmp_power([[[]]], 2)
+    0
+
+    >>> f = ZZ.map([[2], [1, 2, 3]])
+
+    >>> dmp_power(f, 1)
+    [1, 0]
+
+    """
+    pow = []
+    if dmp_zero_p(f, u):
+        return [0]
+    else:
+        for i in range(len(f)):
+            if f[i] != 0:
```

Comment:
May be, AFAICFO, this will return the powers of constants too? Please correct me if I am wrong.

Reply:
No this function will only return power of symbols present in expression. You can go through the tests and can also verify running this PR locally.